runtimes/aleph-alpine-3.13-python/init1.py

```diff
@@ -57,12 +57,12 @@ class Volume:
 
 @dataclass
 class ConfigurationPayload:
-    code: bytes
-    encoding: Encoding
-    entrypoint: str
     input_data: bytes
     interface: Interface
     vm_hash: str
+    code: Optional[bytes] = None
+    encoding: Optional[Encoding] = None
+    entrypoint: Optional[str] = None
```

Comment:
Can we group these fields together in order to make it explicit that they are used for running programs/functions ?